pyproject.toml

```diff
@@ -1,24 +1,37 @@
 [tool.poetry]
 name = "surrealdb"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
To release it to[ PyPi ](https://pypi.org/project/surrealdb/#history) the version can't be "0.2.0" since that was the lastest version that got released 🤔